mmdet3d/datasets/pipelines/loading.py

```diff
@@ -238,6 +240,104 @@ def __repr__(self):
         return f'{self.__class__.__name__}(sweeps_num={self.sweeps_num})'
 
 
+@PIPELINES.register_module()
+class LoadCalibPoseTime(object):
+    """load calib poses and times for SemanticKitti dataset."""
+
+    def __init__(self, use_calib=True, use_pose=True, use_time=False):
+
+        # self.data_root = data_root
+        self.use_calib = use_calib
+        self.use_pose = use_pose
+        self.use_time = use_time
+
+    def __call__(self, results):
+
+        seq_folder = results['ann_info']['pts_semantic_mask_path'].replace(
+            'labels/000000.label', '')
```

Comment:
It's a hard code here. And why only replace 000000.label?

Reply:
I replace the `000000.label` by `sample_idx`.

```
sample_idx = results['sample_idx']
seq_folder = results['ann_info']['pts_semantic_mask_path'].replace(f'labels/{sample_idx}.label', '')
```